src/util/RouterUtil.js

```diff
@@ -124,7 +124,24 @@ function (Okta, Util, OAuth2Util, Enums, BrowserFeatures, Errors, ErrorCodes) {
             }));
           }
         }
-      });
+      };
+
+      if (res.type === Enums.SESSION_STEP_UP && res._embedded.target &&
+          res._embedded.target.type === Enums.TARGET_APP) {
```

Comment:
Do we actually care about checking the target type? Seems like the proposal should work for any target (i.e. you'd only use the target type to configure what to show in the UI).